LEAF_Request_Portal/admin/Group.php

```diff
@@ -162,6 +298,35 @@ public function addMember($member, $groupID)
         }
     }
 
+    /**
+     * [Description for importUser]
+     *
+     * @param string $userID
+     * @param int $groupID
+     * @param string $backupID
+     *
+     * @return array
+     *
+     * Created at: 9/15/2022, 9:30:20 AM (America/New_York)
+     */
+    public function importUser(string $userID, int $groupID, string|null $backupID): array
+    {
+        $this->dataActionLogger->logAction(\DataActions::ADD, \LoggableTypes::EMPLOYEE, [
+            new \LogItem("users", "userID", $userID, $this->getEmployeeDisplay($userID)),
+            new \LogItem("users", "groupID", $groupID, $this->getGroupName($groupID))
+        ]);
+
+        $sql_vars = array(':userID' => $userID,
+                        ':groupID' => $groupID,
+                        ':backupID' => $backupID,);
+
+        $result = $this->db->prepared_query('INSERT INTO users (groupID, userID, backupID)
+                                                    VALUES (:groupID, :userID, :backupID)
+                                                    ON DUPLICATE KEY UPDATE userID=:userID', $sql_vars);
+
+        return $result;
```

Comment:
Going to want to verify this is an array.  Simple solution it to typecast it here.